db/migrate/20220302115803_change_stars_to_be_integer_in_ratings.rb

```diff
@@ -0,0 +1,4 @@
+class ChangeStarsToBeIntegerInRatings < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
You need to add "change_column"
https://guides.rubyonrails.org/active_record_migrations.html